lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -38,6 +38,8 @@ static const StringRef assumeAnnoClass =
     "sifive.enterprise.firrtl.ExtractAssumptionsAnnotation";
 static const StringRef coverAnnoClass =
     "sifive.enterprise.firrtl.ExtractCoverageAnnotation";
+static const StringRef SeqMemAnnoClass =
```

Comment:
Any reason for these not to be const char*?  This does compile to a constant, but so would implicit conversion of the string at the call site.

Reply:
I think the best way to do this is actually `static const char SeqMemAnnoClass[] = "foo";`.  That avoids a relocatable pointer + array and just stores an array.